api/predict.py

```diff
@@ -19,13 +19,13 @@
 from flask_restplus import fields
 from werkzeug.datastructures import FileStorage
 
-# Set up parser for input data (http://flask-restplus.readthedocs.io/en/stable/parsing.html)
+# Set up parser for input data <http://flask-restx.readthedocs.io/en/stable/parsing.html>
```

Comment:
Did you see the note when you go to this URL?

> The whole request parser part of Flask-RESTX is slated for removal and will be replaced by documentation on how to integrate with other packages that do the input/output stuff better (such as marshmallow). This means that it will be maintained until 2.0 but consider it deprecated. Don’t worry, if you have code using that now and wish to continue doing so, it’s not going to go away any time too soon.